.github/workflows/delete-test-reports.yml

```diff
@@ -0,0 +1,26 @@
+name: Delete Test Report
+
+on:
+  pull_request:
+    types: [closed]
+
+env:
+  BRANCH_NAME: ${{ github.event.pull_request.head.ref }}
+
+jobs:
+  delete-report:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Node
+        uses: ./.github/actions/node
+
+      - name: Delete Report
+        run: |
+          echo "Deleting report for branch $BRANCH_NAME"
+          BRANCH_URL=$(echo ${BRANCH_NAME} | sed -e 's/[^a-zA-Z0-9]/-/g')
+          DOMAIN=https://kiwicom-orbit-test-report-${BRANCH_URL}.surge.sh
+          yarn surge teardown $DOMAIN --token ${{ secrets.SURGE_TOKEN }}
+        continue-on-error: true
```

Comment:
Why do we need this?

Reply:
I didn't want the pipeline to be red if it fails on deleting (because it doesn't exist, for instance).
Imagine a PR that never fails the tests. There won't be reports. But this job will still execute and try to delete… nothing. The command above will fail, but the pipeline won't.

I found it easier to do than checking if the report exists before deleting it.

Reply:
Oh I see, I hadn't considered that case.